app/helpers/notifier_helper.rb

```diff
@@ -2,4 +2,13 @@ module NotifierHelper
   def fp(text)
     format_paragraph(text, 72, 0)
   end
+
+  def link_to_user(display_name)
+    link_to(
+      display_name,
+      user_url(display_name, :host => SERVER_URL),
+      :target => "_blank",
+      :style => "text-decoration: none; color: #222; font-weight: bold"
+    )
+  end
 end
```

Comment:
I suppose the alternative would be not to create a helper, and copy-paste its contents in the templates instead of the call to the helper. Inline the helper function, basically. But if you're happy enough with this, I'm happy too.